[BUG]: abp fil benchmark broken

### Version

22.11

### Which installation method(s) does this occur on?

Source

### Describe the bug.

```
FAILED test_bench_e2e_pipelines.py::test_abp_fil_e2e - AssertionError: Number of features in preprocessing 18, does not match configuration 29
```

### Minimum reproducible example

```shell
cd tests/benchmarks
pytest -s --run_benchmark --benchmark-enable --benchmark-warmup=on --benchmark-warmup-iterations=1 --benchmark-autosave test_bench_e2e_pipelines.py
```


### Relevant log output

```
================================================================================= FAILURES =================================================================================
_____________________________________________________________________________ test_abp_fil_e2e _____________________________________________________________________________

benchmark = <pytest_benchmark.fixture.BenchmarkFixture object at 0x7fb000021e40>, tmp_path = PosixPath('/tmp/pytest-of-dagardner/pytest-12/test_abp_fil_e2e0')

    @pytest.mark.benchmark
    def test_abp_fil_e2e(benchmark, tmp_path):
    
        config = Config()
        config.mode = PipelineModes.FIL
        config.num_threads = E2E_TEST_CONFIGS["test_abp_fil_e2e"]["num_threads"]
        config.pipeline_batch_size = E2E_TEST_CONFIGS["test_abp_fil_e2e"]["pipeline_batch_size"]
        config.model_max_batch_size = E2E_TEST_CONFIGS["test_abp_fil_e2e"]["model_max_batch_size"]
        config.feature_length = E2E_TEST_CONFIGS["test_abp_fil_e2e"]["feature_length"]
        config.edge_buffer_size = E2E_TEST_CONFIGS["test_abp_fil_e2e"]["edge_buffer_size"]
        config.class_labels = ["mining"]
        config.fil = ConfigFIL()
        fil_cols_filepath = os.path.join(TEST_DIRS.data_dir, 'columns_fil.txt')
        config.fil.feature_columns = load_labels_file(fil_cols_filepath)
        CppConfig.set_should_use_cpp(True)
    
        input_filepath = E2E_TEST_CONFIGS["test_abp_fil_e2e"]["file_path"]
        repeat = E2E_TEST_CONFIGS["test_abp_fil_e2e"]["repeat"]
        output_filepath = os.path.join(tmp_path, "abp_fil_e2e_output.csv")
        model_name = "abp-nvsmi-xgb"
    
>       benchmark(fil_pipeline, config, input_filepath, repeat, output_filepath, model_name)

test_bench_e2e_pipelines.py:176: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../../../conda/envs/m2/lib/python3.10/site-packages/pytest_benchmark/fixture.py:125: in __call__
    return self._raw(function_to_benchmark, *args, **kwargs)
../../../conda/envs/m2/lib/python3.10/site-packages/pytest_benchmark/fixture.py:147: in _raw
    duration, iterations, loops_range = self._calibrate_timer(runner)
../../../conda/envs/m2/lib/python3.10/site-packages/pytest_benchmark/fixture.py:275: in _calibrate_timer
    duration = runner(loops_range)
../../../conda/envs/m2/lib/python3.10/site-packages/pytest_benchmark/fixture.py:90: in runner
    function_to_benchmark(*args, **kwargs)
test_bench_e2e_pipelines.py:85: in fil_pipeline
    pipeline.add_stage(PreprocessFILStage(config))
../../morpheus/pipeline/stream_wrapper.py:45: in inner
    func(self, *args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <preprocess-fil-74; PreprocessFILStage()>
c = Config(debug=False, log_level=30, log_config_file=None, plugins=None, mode=<PipelineModes.FIL: 'FIL'>, feature_length=....gpu.max_clocks.graphics_clock', 'nvidia_smi_log.gpu.max_clocks.sm_clock', 'nvidia_smi_log.gpu.max_clocks.mem_clock']))

    def __init__(self, c: Config):
        super().__init__(c)
    
        self._fea_length = c.feature_length
        self.features = c.fil.feature_columns
    
>       assert self._fea_length == len(self.features), \
            f"Number of features in preprocessing {len(self.features)}, does not match configuration {self._fea_length}"
E       AssertionError: Number of features in preprocessing 18, does not match configuration 29

../../morpheus/stages/preprocess/preprocess_fil_stage.py:57: AssertionError
```

### Full env printout

_No response_

### Other/Misc.

_No response_

### Code of Conduct

- [X] I agree to follow Morpheus' Code of Conduct
- [X] I have searched the [open bugs](https://github.com/nv-morpheus/Morpheus/issues?q=is%3Aopen+is%3Aissue+label%3Abug) and have found no duplicates for this bug report

--- a/tests/benchmarks/e2e_test_configs.json
+++ b/tests/benchmarks/e2e_test_configs.json
@@ -15,7 +15,7 @@
         "num_threads": 8,
         "pipeline_batch_size": 1024,
         "model_max_batch_size": 1024,
-        "feature_length": 29,
+        "feature_length": 18,
         "edge_buffer_size": 4
     },
     "test_phishing_nlp_e2e": {

